app/lib/services/import.dart

```diff
@@ -511,15 +511,13 @@ class ImportService {
         bytes,
         dpi: dpi,
       )) {
+        img.Image image = page.asImage();
```

Comment:
please use final if the value doesn't change
```suggestion
        final image = page.asImage();
```